src/api/lib/rubocop/cop/view_component/file_name.rb

```diff
@@ -0,0 +1,20 @@
+module RuboCop
```

Comment:
Inspired by [upstream](https://github.com/rubocop/rubocop/blob/b806ff29f10702401ffd9ea3fd7d5a2d10391215/lib/rubocop/cop/naming/file_name.rb), but this is much simpler here :slightly_smiling_face: